src/Pages/RegistrationForm/RegistrationForm.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from "react";
+import Button from "../../components/Button";
+import Input from "../../components/Input";
+import "./registrationForm.css";
+import { RegiFormProps } from "./types";
+
+const RegistrationForm = () => {
+  const [password, setPassword] = useState<string>("");
+  const [confirmPassword, setConfirmPassword] = useState<string>("");
+  const [error, setError] = useState<string>("");
+
+  const handlePasswordChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setPassword(e.target.value);
+  };
+  const handleConfirmPasswordChange = (
+    e: React.ChangeEvent<HTMLInputElement>
+  ) => {
+    setConfirmPassword(e.target.value);
+  };
+
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    if (password != confirmPassword) {
+      setError("Passwords do not match.");
+    } else {
+      setError("");
+    }
+  };
+
+  const handleReset = () => {
+    setPassword("");
+    setConfirmPassword("");
+    setError("");
+  };
+  return (
+    <>
+      <h1 className="heading">HELLO USER </h1>
```

Comment:
design is different

<img width="380" alt="Screenshot 2024-10-31 at 2 24 33 PM" src="https://github.com/user-attachments/assets/fbb55ac7-85b9-4145-821b-67e6f33d8490">